fs_xgspon_mod.py

```diff
@@ -771,6 +795,12 @@ def parse_vlan_filter(vf):
     parse_rearm.add_argument("serial", type=parse_serial)
     parse_rearm.set_defaults(func=rearm)
 
+    parse_overrideslot = s.add_parser("overrideslot")
+    parse_overrideslot.add_argument("--onu_ip", default="192.168.100.1")
+    parse_overrideslot.add_argument("fs_onu_serial", type=parse_serial)
+    parse_overrideslot.add_argument("eth_slot", type=int, choices=(range(1, 65534)), metavar="[1-65534]")
```

Comment:
Unfortunately, the slots need to be particularly constrained. Some of them have hardcoded purposes (can be found in both the `MecMgr` and `EthMgr` binaries) and while it's been a while if I remember it is used directly as an array index in some places where too-high values would cause a crash. 1 and 10 are known safe values, and I'd need to confirm manually what other values are safe (but I believe all safe values are between 1 and 10).